tools/alps/src/Makefile.am

```diff
@@ -41,20 +41,26 @@ AM_CPPFLAGS = \
   -I$(abs_top_srcdir)/launchmon/src \
   -I$(abs_top_srcdir)/@LMONAPILOC@ 
 
-bin_PROGRAMS = alps_be_starter alps_fe_colocator 
-alps_be_starter_SOURCES = alps_be_starter.c 
+INCLUDES                          = -I$(top_srcdir) \
+				    -I$(top_srcdir)/launchmon/src \
+				    -I$(top_srcdir)/@LMONAPILOC@
+
+bin_PROGRAMS              = alps_be_starter alps_fe_colocator 
```

Comment:
I don't think there are any changes in those two lines unless I'm missing something?